js/switch.js

```diff
@@ -21,6 +21,7 @@ function setDeviceSwitch(idx, status) {
     /* Check if switch exists --> create or update */
     if (switcher.length == 0) {
         $(tr).find("#status").hide();
+        $(tr).find("#bigtext").hide();
```

Comment:
You should use , (comma) instead of duplicate lines --> https://api.jquery.com/multiple-selector/